application/controllers/changePassword.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require "./vendor/autoload.php";
+session_start();
```

Comment:
Why start session here?

Reply:
Sir in this file I am just checking whether the user is valid or not like if case user is not valid and try to enter the page just type the URL in the address bar, in that case, I use the session to verify the details if a user is not valid I immediately destroy the session and redirect to the login in page.